pkg/scorecard.go

```diff
@@ -95,6 +95,7 @@ func RunScorecard(ctx context.Context,
 	commitSHA string,
 	commitDepth int,
 	checksToRun checker.CheckNameToFnMap,
+	probesToRun []string,
```

Comment:
for right now, is there anyway to do the filtering without modifying this entrypoint? This is a breaking change in the most externally used function.

Haven't looked into it too much, but would we be able to build a list of checks to run from the requested probes, then filter it on the results side later?

Reply:
An easy work-around would be to implement a `RunScorecardProbes` function which is the same as `RunScorecard` but with the probe argument.

Reply:
something like:
`ExperimentalRunProbes`

Reply:
Changed and rebased